lib/jxl/encode.cc

```diff
@@ -387,17 +387,20 @@ JxlEncoderStatus JxlEncoderStruct::RefillOutputByteQueue() {
       ib.origin.x0 = input_frame->option_values.header.layer_info.crop_x0;
       ib.origin.y0 = input_frame->option_values.header.layer_info.crop_y0;
     }
+    JXL_ASSERT(writer.BitsWritten() == 0);
     if (!jxl::EncodeFrame(input_frame->option_values.cparams, frame_info,
                           &metadata, input_frame->frame, &enc_state, cms,
                           thread_pool.get(), &writer,
                           /*aux_out=*/nullptr)) {
       return JXL_API_ERROR("Failed to encode frame");
     }
+    codestream_bytes_written_beginning_of_frame =
+        codestream_bytes_written_end_of_frame;
+    codestream_bytes_written_end_of_frame += (writer.BitsWritten() + 7) / 8;
```

Comment:
nit: could use jxl::DivCeil(writer.BitsWritten(), 8) here since it's the convention in the rest of the code

Reply:
fixed